### ISSUE TITLE
Date picker is not showing when the picker is inside the modal

### ISSUE THREAD

**Issue by everestster** [henninghall/react-native-date-picker#398]
Hi,

I have the modal that open the item details on clicking on the list of items. The picker works fine without the modal but I'd like to show the date picker in the modal. It works perfectly fine on Android. However, on IOS, I'm getting the following error.

`Attempt to present <UIAlertController: 0x1049cd800> on <UIViewController: 0x1045121a0> (from <UIViewController: 0x1045121a0>) which is already presenting <RCTModalHostViewController: 0x11a09a720>.`

To replicate
"react": "17.0.2",
"react-native": "^0.65.1",
"react-native-date-picker": "^4.0.0",

I use import {Modal} from 'react-native';

**Comment by pratius** [comment]
Same Issues.

**Comment by JDJBSPL** [comment]
Having same issue.

**Comment by marg51** [comment]
yeah the DatePicker's modal mode doesn't work inside a react native Modal. 

```
"react-native": "~0.63.4",
"react-native-date-picker": "^4.0.0",
"react": "16.13.1",
```


Found that online, which seems related: https://stackoverflow.com/questions/32696615/warning-attempt-to-present-on-which-is-already-presenting-null

but I don't know how to try the proposed solution 🙃

**Comment by henninghall** [comment]
This should now have been solved in `v4.1.3`, please try it out and let me know if it fixed it for you 🙂
